lib/command/command.js

```diff
@@ -65,7 +65,16 @@ externals.Commands = class {
   validate (slackResponse) {
     return Promise.resolve({
       then: (onFulfill, onReject) => {
-        if (!internals.isCommandAllowed(this.getCommand(),
+        if (!internals.isAllowedChannel(this.getCommand(), slackResponse,
+            this.getSlackData())) {
+          /* jshint ignore:start */
+          return onReject({
+            allowed_channels: true,
```

Comment:
This is going to be used in template. From the context of the template it makes sense to call it as `restricted_cahnnels` like here https://github.com/usubram/slack-quick-bots/blob/master/lib/template/error.hbs#L9